addons/MultiAudio/src/presenter.js

```diff
@@ -14,17 +14,71 @@ function AddonMultiAudio_create(){
     presenter.playerController = null;
```

Comment:
There is no upgrade model function

Reply:
There are no new properties, just a values in already existing ones, so there shouldn't be any problems.